plugins/sqlfluff-templater-dbt/test/templater_test.py

```diff
@@ -421,7 +422,8 @@ def test__templater_dbt_handle_database_connection_failure(
 def test__project_dir_does_not_exist_error(dbt_templater, caplog):  # noqa: F811
     """Test an error is logged if the given dbt project directory doesn't exist."""
     dbt_templater.sqlfluff_config = FluffConfig(
-        configs={"templater": {"dbt": {"project_dir": "./non_existing_directory"}}}
+        configs={"templater": {"dbt": {"project_dir": "./non_existing_directory"}}},
+        overrides={"dialect": "ansi"},
```

Comment:
Why is one ` configs` and the other an `overrides` out of interest? Is this cause a default `templater` exists but a default `dialect` no longer does now?

Reply:
Based on this line in the `FluffConfig` constructor, I'm pretty sure either would work:
```
        self._configs = nested_combine(
            defaults, configs or {"core": {}}, {"core": overrides or {}}
        )
```

Once I found a way to get tests passing, I did a lot of copy paste. I'll look at changing this for simplicity.

Reply:
Yes, that worked. Updating a couple of dbt tests.